pkg/platform/kube/functionres/lazy.go

```diff
@@ -2160,18 +2161,24 @@ func (lc *lazyClient) getMetricResourceByName(resourceName string) v1.ResourceNa
 	}
 }
 
-func (lc *lazyClient) resolveFailFast(pods []v1.Pod,
+func (lc *lazyClient) resolveFailFast(podsList *v1.PodList,
+	namespace string,
```

Comment:
why? pod has namespace on its `.Meta` class. will the `namespace` here be different?